src/Services/Basket/Basket.API/Auth/Server/AuthorizationHeaderParameterOperationFilter.cs

```diff
@@ -1,33 +1,26 @@
-﻿using Microsoft.AspNetCore.Mvc.Authorization;
-using Microsoft.OpenApi.Models;
-using Swashbuckle.AspNetCore.SwaggerGen;
-using System.Collections.Generic;
-using System.Linq;
+﻿namespace Microsoft.eShopOnContainers.Services.Basket.API.Auth.Server;
 
-namespace Microsoft.eShopOnContainers.Services.Basket.API.Auth.Server
+public class AuthorizationHeaderParameterOperationFilter : IOperationFilter
 {
-    public class AuthorizationHeaderParameterOperationFilter : IOperationFilter
+    public void Apply(OpenApiOperation operation, OperationFilterContext context)
     {
-        public void Apply(OpenApiOperation operation, OperationFilterContext context)
-        {
-            var filterPipeline = context.ApiDescription.ActionDescriptor.FilterDescriptors;
-            var isAuthorized = filterPipeline.Select(filterInfo => filterInfo.Filter).Any(filter => filter is AuthorizeFilter);
-            var allowAnonymous = filterPipeline.Select(filterInfo => filterInfo.Filter).Any(filter => filter is IAllowAnonymousFilter);
+        var filterPipeline = context.ApiDescription.ActionDescriptor.FilterDescriptors;
+        var isAuthorized = filterPipeline.Select(filterInfo => filterInfo.Filter).Any(filter => filter is AuthorizeFilter);
+        var allowAnonymous = filterPipeline.Select(filterInfo => filterInfo.Filter).Any(filter => filter is IAllowAnonymousFilter);
 
-            if (isAuthorized && !allowAnonymous)
-            {
-                if (operation.Parameters == null)
-                    operation.Parameters = new List<OpenApiParameter>();
+        if (isAuthorized && !allowAnonymous)
+        {
+            if (operation.Parameters == null)
+                operation.Parameters = new List<OpenApiParameter>();
```

Comment:
```suggestion
            operation.Parameters ??= new List<OpenApiParameter>();
```

I believe that you simply do this instead. It will only assign a new list if it's `null`.